lib/build.sh

```diff
@@ -165,6 +165,7 @@ pushd "${CHECKOUT}/libvips-${VIPS_VERSION}-${TARGET}"
      --without-magick \
      --without-orc \
      --without-gsf \
+     --without-rsvg \
```

Comment:
Do we need it?

Reply:
By default, libvips configure script will test if libsvg is present on the system. If it is, libsvg is added as dependency. Libsvg also depends on libpng. It links with the installed one on the system and then compiler / link flags are spread in the VIPS_INCLUDE variable.
Then, your build is broken because libvips will be compiled with libpng12 and JVips load libpng16.